anaconda_server/commands/doc.py

```diff
@@ -5,13 +5,23 @@
 import sys
 import logging
 
-try:
+# Old approach:
+# try:
+#     import html
+# except ImportError:
+#     # python2 faillback
+#     import cgi
+#     from HTMLParser import HTMLParser
+
+# New approach:
+if sys.version_info >= (3, 0):
```

Comment:
Please, remove all those commented lines, no one would care about the old approach but add a comment saying that we are forced to use a non Pythonic import approach here as the incomplete module `future.moves.html` from PyCharmers breaks the `doc.py` logic so we are forced to use these conditionals.